forge/test/mlir/llama/tests/test_specific_ops_llama32.py

```diff
@@ -0,0 +1,480 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+import os
+import pytest
+
+import torch
+from torch import nn
+
+import forge
+from forge.op.eval.common import compare_with_golden_pcc, compare_with_golden
+
+
+@pytest.mark.parametrize(
+    "shapes",
+    [
+        [(1, 11, 1), (1,)],
+        [(1, 32, 11, 64), (1, 32, 11, 64)],
+        [(1, 8, 11, 64), (1, 8, 11, 64)],
+        [(1, 32, 11, 11), (1, 1, 11, 11)],
+        [(1, 11, 2048), (1, 11, 2048)],
+    ],
+)
+def test_add(shapes):
+
+    if shapes[0] != shapes[1]:
+        pytest.xfail("eltwise_add broadcast not supported")
+
+    class Add(nn.Module):
+        def __init__(self):
+            super().__init__()
+
+        def forward(self, a, b):
+            return a + b
+
+    inputs = [torch.rand(shapes[0]), torch.rand(shapes[1])]
+
+    framework_model = Add()
+    fw_out = framework_model(*inputs)
+
+    compiled_model = forge.compile(framework_model, sample_inputs=inputs)
+    co_out = compiled_model(*inputs)
+
+    co_out = [co.to("cpu") for co in co_out]
+    fw_out = [fw_out] if isinstance(fw_out, torch.Tensor) else fw_out
+    assert all([compare_with_golden_pcc(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
+
+
+@pytest.mark.parametrize(
+    "inputs_and_dim",
+    [
+        ((1, 11, 32), (1, 11, 32), -1),
+        ((1, 32, 11, 32), (1, 32, 11, 32), -1),
+        ((1, 8, 11, 32), (1, 8, 11, 32), -1),
+    ],
+)
+def test_concat(inputs_and_dim):
+    in_shape1, in_shape2, dim = inputs_and_dim
+    print(in_shape1, in_shape2, dim)
```

Comment:
Let's remove these prints, seems redundant as we can get shape info from test logs as well :))

Reply:
Oops, i left those from testing, removed now